oak_runtime/src/introspection_browser_client/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
```

Comment:
```suggestion
#!/bin/bash
```

https://google.github.io/styleguide/shellguide.html#which-shell-to-use